File: we_10_1_inertial_drift_during_a_gnss_outage.py
# Integrated Navigation Systems — Chapter 10: GNSS/INS Integration Architectures
# Worked Example 10.1 — Inertial drift during a GNSS outage
# Companion code (educational reference implementation; see the book for context).

ba,bg,g,t=0.01,4.8e-6,9.81,30
print(f'accel={0.5*ba*t**2:.1f} m  gyro={(1/6)*bg*g*t**3:.2f} m')  # 4.5 0.21

File: we_10_1_inertial_drift_during_a_gnss_outage.m
% Integrated Navigation Systems — Chapter 10: GNSS/INS Integration Architectures
% Worked Example 10.1 — Inertial drift during a GNSS outage
% Companion code (educational reference implementation; see the book for context).

ba=0.01; bg=4.8e-6; g=9.81; t=30;
dpa=0.5*ba*t^2; dpg=(1/6)*bg*g*t^3;
fprintf('accel=%.1f m  gyro=%.2f m\n', dpa, dpg);   % 4.5 0.21
